### PR TITLE
Feature/faq

### PR DESCRIPTION
**Pull request by rouzbeh84**
Very beginner and high level of questions. Just want to get the ball rolling on this to get some feedback from the core team.

### CODE REVIEW

**Comment by rouzbeh84** [comment]
Should I make an issue to let everyone add their FAQ suggested questions/answers or let it run here until we get to minimum needed?

File: public/faq.jade [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    h3 How can I participate in an event as a Speaker/Host/Sponsor?
+    p If you have to ask you'll never know. Just kidding! We have a page dedicated specifically for this and especially for you! Visit <a href="http://contribute.js.la/">contribute.js.la</a> and choose the appropriate option to fill out a short form to get started. Someone will reach out to you very soon afterwards and thank you in advance for the inerest!
+
+    h3 Is there a code of conduct?
+    p We're so glad you asked! We absolutely have a CoC so as to make everyone feel very welcome and inclusive of meeting new people and making new friends in the name of the almighty JavaScript Overlords. It can be found <a href="">here</a>
```
Comment (jcblw) [inline public/faq.jade]: link to here goes no where

**Comment by jcblw** [comment]
looks good, I would say lets merge and not sit on it. I'm currently setting up a ci deploy so it will auto deploy on merge. Also whats going on with the svg? can we remove that change?

**Comment by rouzbeh84** [comment]
i just rebased and thought i dropped the svg commit. I'll double check and fix it now

**Comment by rouzbeh84** [comment]
should be good to go now

**Comment by jcblw** [comment]
its up http://js.la/faq

**Pull request merged**